### PR TITLE
changelog for 0123

### PR DESCRIPTION
**Pull request by MuyangAmigo**

### CODE REVIEW

File: packages/vscode-extension/PRERELEASE.md [hunk 1]
```diff
@@ -2,6 +2,56 @@
 
 ## Changelog
 
+### January 23, 2024
+
+#### New Features
+
+- **Deploy Tab Apps to Static Web App**: Azure Static Web Apps, an automatic service for building and deploying full-stack web apps to Azure from a code repository, is now the default solution for deploying Tab-based applications in Teams Toolkit.
```
Comment (therealjohn) [inline packages/vscode-extension/PRERELEASE.md:9]: Is there a way to use the old way, or a sample that we can link if the user prefers it?

File: packages/vscode-extension/PRERELEASE.md [hunk 2]
```diff
@@ -2,6 +2,56 @@
...
+
+### Enhancements
+
+- **Clean up `.deployment` Folder in Between Deployments**: Teams Toolkit now cleans up the `.deployment` folder in the build directory before each deployment, addressing a [known issue](https://github.com/OfficeDev/TeamsFx/issues/10075) and reducing deployment time.
```
Comment (therealjohn) [inline packages/vscode-extension/PRERELEASE.md:13]: Between can be lower case.

**Review by therealjohn — CHANGES_REQUESTED** [review]

File: packages/vscode-extension/PRERELEASE.md (hunk 2 above)
Comment (MuyangAmigo) [reply packages/vscode-extension/PRERELEASE.md:13]: Updated

File: packages/vscode-extension/PRERELEASE.md (hunk 1 above)
Comment (MuyangAmigo) [reply packages/vscode-extension/PRERELEASE.md:9]: Linked to a sample

File: packages/vscode-extension/PRERELEASE.md [hunk 3]
```diff
@@ -2,6 +2,56 @@
...
+#### New Additions to the Sample App Gallery
+
+- **Format Reddit Link into Adaptive Card**: This sample application demonstrates how to format a Reddit link into an Adaptive Card in Microsoft Teams conversations.
+![Link Unfurling Sample](https://github.com/OfficeDev/TeamsFx/assets/11220663/0d44f8c3-d02e-4912-bfa2-6ed3fdb29c1b)
+
+#### Teams Toolkit CLI ([`@microsoft/teamsapp-cli`](https://www.npmjs.com/package/@microsoft/teamsapp-cli)) `v3.0.0@beta`
+![Teams Toolkit CLI](https://camo.githubusercontent.com/67608a468cbd406d6ff18585c8bc3b34d3d97d0a8ef525bdf516ca23fd5e32dd/68747470733a2f2f616b612e6d732f636c692d6865726f2d696d616765)
+Teams Toolkit CLI version 3 is now in public preview. Major changes include:
+
+- **New Command Signature**: Teams Toolkit CLI now starts with `teamsapp` as the root command signature for more clarity. We recommend changing your scripts to use `teamsapp` as the command prefix.
```
Comment (sffamily) [inline packages/vscode-extension/PRERELEASE.md:35]: need to give a clear message when the old signature is no longer supported. or to call out the previous signature is still working.

**Review by therealjohn — APPROVED** [review]

File: packages/vscode-extension/PRERELEASE.md (hunk 3 above)
Comment (MuyangAmigo) [reply packages/vscode-extension/PRERELEASE.md:35]: Sure, we will share the announcement when we release v3 stable version. v2 will be supported for another year after v3 is released.

**Pull request merged**